PCbuild/pyproject.props

```diff
@@ -98,10 +98,10 @@
           Inputs="$(PySourcePath)Include\patchlevel.h"
           Outputs="$(IntDir)pythonnt_rc.h">
     <WriteLinesToFile File="$(IntDir)pythonnt_rc.h" Overwrite="true" Encoding="ascii"
-                      Lines='/* This file created by pyproject.props /t:GeneratePythonNtRcH */
-#define FIELD3 $(Field3Value)
-#define MS_DLL_ID "$(SysWinVer)"
-#define PYTHON_DLL_NAME "$(TargetName)$(TargetExt)"
+                      Lines='/* This file created by pyproject.props /t:GeneratePythonNtRcH */%0D%0A
```

Comment:
After I installed VS 2019 to test something else this wouldn't build anymore without the %0D%0A after each line.  I reported the issue.  Let me know if you want to do something different here.